ffmpeg_plugin/0001-lavc-svt_hevc-add-libsvt-hevc-encoder-wrapper.patch

```diff
@@ -407,8 +408,8 @@ index 0000000..d9ac04c
 +            header_ptr->nAllocLen = 0;
 +            header_ptr->nFilledLen = 0;
 +            header_ptr->nTickCount = 0;
-+            header_ptr->pBuffer = NULL;
 +            header_ptr->nFlags = EB_BUFFERFLAG_EOS;
++            header_ptr->pBuffer = NULL;
```

Comment:
@tianjunwork Please check if GST has similar memleak.
SVT HEVC Lib asks us to set the input buffer's pBuffer to NULL, while we need to free that when closing.

Reply:
Sure, will do. Thanks for reminding.